lib/webrtc/rtcpeerconnection/chrome.js

```diff
@@ -104,6 +104,11 @@ function ChromeRTCPeerConnection(configuration) {
     }
   });
 
+  peerConnection.ontrack = function ontrack() {
+    // NOTE(mroberts): adapter.js's "track" event shim only kicks off if we set
```

Comment:
@markandrus Nitpick: Can we move this comment above the assignment to `ontrack`? Usually comments inside the function describe implementation.
```javascript
// NOTE(mroberts): ...
peerConnection.ontrack = function ontrack() {};
```